components/brave_new_tab_ui/components/default/settings/index.ts

```diff
@@ -452,3 +452,83 @@ export const ToggleCardsSwitch = styled<{}, 'div'>('div')`
   float: right;
   margin: -65px -10px 0 0;
 `
+
+export const StyledTopSitesCustomizationSettings = styled('div')`
+  --widget-gap: 10px;
+  font-family: ${p => p.theme.fontFamily.heading};
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-top: 16px;
+  gap: var(--widget-gap)
+`
+
+export const StyledTopSitesCustomizationSettingsOption = styled<{}, 'div'>('div')`
+  width: calc(50% - var(--widget-gap) / 2);
+  display: flex;
+  flex-direction: column;
+`
+
+interface CustomizationImageBorderProps {
+  selected: boolean
+}
+
+export const StyledTopSitesCustomizationImageBorder = styled<CustomizationImageBorderProps, 'div'>('div')`
+  margin-bottom: 8px;
+  border-radius: 11px;
+
+  ${p => p.selected && css`
+    background: ${p => isDarkTheme(p) ? 'linear-gradient(314.42deg, #FA7250 6.04%, #FF1893 44.31%, #A78AFF 100%)'
+                                      : 'linear-gradient(122.53deg, #4C54D2 0%, #BF14A2 56.25%, #F73A1C 100%)'};
+    padding: 3px;
+  `}
+
+  ${p => !p.selected && css`
+    border: 2px solid ${p => isDarkTheme(p) ? '#3B3E4F' : '#F1F3F5'};
+    background: ${p => isDarkTheme(p) ? '#17171F' : '#FFF'};
+    padding: 1px;
+
+    &:hover {
+      border: 2px solid ${p => isDarkTheme(p) ? '#C2C4CF' : '#AEB1C2'};
+      background: ${p => isDarkTheme(p) ? '#2B2D3F' : '#FAFAFF'};
+      padding: 1px;
+    }
+
+    &:focus {
+      border: 3px solid rgba(251, 84, 43, 0.6);
+      background: ${p => isDarkTheme(p) ? '#2B2D3F' : '#FAFAFF'};
+      padding: 0px;
+    }
```

Comment:
Since this element is not focusable, I don't think this code is doing anything. If it does work or you move it to an element that is focusable, please use `focus-visible` so that it only shows for non-mouse users.